Tasks/Common/VstsAzureHelpers_/VstsAzureHelpers_.psm1

```diff
@@ -51,5 +51,41 @@ function Initialize-Azure {
     }
 }
 
+function Get-AzureCmdletsVersion
```

Comment:
I see now that AzureUtility.ps1 uses this. Please do not add this function. *I think* AzureRM.profile is the module you are after. By the time the Azure helpers are initialized, the AzureRM.profile module will be imported. If comparison against that module is what you are after, then the correct way to compare version is:
`if (Get-Module -Name AzureRM.profile).Version -lt ([version]'1.2.3'))`

You should not use `-ListAvailable` and you should not use `AzureRM*`

The AzureHelpers module prefers importing from the PSModulePath and then falls back to the installed SDK path (opposite of previous Azure PowerShell). When it's imported from the SDK, only AzureRM.profile is explicitly imported. When it's imported from the PSModule path, AzureRM and AzureRM.profile are imported. The AzureRM module is uninteresting, it's main purpose is simply to install the rest of the AzureRM modules, and it's version number does not even match the rest of the AzureRM modules that it's used to install.